api_rest/mvnw

```diff
@@ -0,0 +1,316 @@
+#!/bin/sh
```

Comment:
do we need this?

Reply:
Good morning, yes that's right. the reason is beacause is a spring maven project.